src/MCMCChains.jl

```diff
@@ -24,6 +24,8 @@ import Random
 import Serialization
 import Statistics: std, cor, mean, var, mean!
 
+import StatsModelComparisons: dic
+
```

Comment:
I'd prefer if `dic` is not imported but qualified when implemented below:
```suggestion
import StatsModelComparisons

```